lib/helpers/utils.dart

```diff
@@ -82,6 +83,10 @@ Future<dynamic>? handleSectionItemClick(BuildContext context, Fragment$ItemSecti
   final router = context.router;
   var episodeItem = item.asOrNull<Fragment$ItemSectionItem$item$$Episode>();
   if (episodeItem != null) {
+    final curLiveEpisodeId = ref.read(currentLiveEpisodeProvider).valueOrNull?.episode?.id;
+    if (curLiveEpisodeId == episodeItem.id) {
+      return overrideAwareNavigation(navigationOverride, router, const LiveScreenRoute());
```

Comment:
I'm a bit unsure how this interacts with the `pushInsteadOfReplace` override.. pushInsteadOfReplace is used on standalones, so can you just double check that it works smooth with e.g. https://app.bcc.media/episode/1361 ?

Reply:
It works fine with that episode. But to me it looks like the same code is used to handle all episode taps.

Reply:
can you just remove overrideAwareNavigation anyway? it's never relevant for the live route.

Reply:
> can you just remove overrideAwareNavigation anyway? it's never relevant for the live route.

Got it. 👍